Access Engineering.js

```diff
@@ -0,0 +1,126 @@
+{
+	"translatorID": "d120a8a7-9d45-446e-8c18-ad9ef0a6bf47",
+	"label": "Access Engineering",
+	"creator": "Vinoth K - highwirepress.com",
+	"target": "^https://www.accessengineeringlibrary.com/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2022-04-01 14:06:45"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020-2021 Vinoth K - highwirepress.com
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	let title = attr(doc, 'meta[name="citation_title"]', 'content');
+	if (title) {
+		if (doc.querySelector('meta[name="citation_isbn"]')) {
+			let bookTitle = attr(doc, 'meta[name="citation_book_title"]', 'content');
+			if (!bookTitle && title != bookTitle) {
+				return "book";
+			} else {
+				return "bookSection";
+			}
+		} else {
+			return "journalArticle";
+		}
+	}
+	return false;
+}
+
+function doWeb(doc, url) {
+	scrape(doc, url);
```

Comment:
if/when adding multiples, you'd need to add the relevant logic here (look at other translators for templates)

Reply:
Handled multiples and based that modified script and also attached test cases for the same. Kindly check it.